src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts.mdx

```diff
@@ -0,0 +1,241 @@
+
+Adapt the `config.yaml` with these recommended parameters:
+
+<Callout variant="important">
+CPU, load, memory and disk utilization metrics require otelcol-contri release `v47` or greater.
+</Callout>
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "290px" }}>
+        Configuration
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        `receivers::hostmetrics`
+      </td>
+      <td>
+        Enable host metrics, 20 seconds interval is recommended (same default as the Infrastructure Agent), it should not be greater than 60 seconds to avoid issues with “host not responding” alerts. Process instrumentation is optional.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::resourcedetection`
+      </td>
+      <td>
+        Keep the following in mind:
+
+          * `env`: Reads resource information from the `OTEL_RESOURCE_ATTRIBUTES` environment variable.
+          * `system`: Adds `host.name` and `os.type`.
+          * For cloud environments, a specific resource detection processor needs to be configured so that metrics are decorated with `host.id` (required to identify the Host entity in New Relic One). Common cloud detectors are `gce` for GCP machines, `ec2` for AWS EC2 and `azure` for Azure VMs. Additional processors are available for orchestrated environments.
+          * For on-premises systems (or unsupported cloud environments), a `host.id` attribute is required. The `resource` processor can be used to copy the `host.name` value (from `system`) as a new `host.id` attribute. Note this value should be unique across all instrumented hosts:
+          ```
+          resource:
+              attributes:
+                - key: host.id
+                  from_attribute: host.name
+                  action: upsert
+          ```
+
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::batch`
+      </td>
+      <td>
+        The [batch processor](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md) accepts spans, metrics, or logs and places them into batches. Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data. This processor supports both size and time based batching.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::memory_limited`
+      </td>
+      <td>
+        The [memory limiter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor) processor is used to prevent out of memory situations on the collector. Given that the amount and type of data the collector processes is environment specific and resource utilization of the collector is also dependent on the configured processors, it is important to put checks in place regarding memory usage.
```

Comment:
If this was my doc, I might reword last paragraph to something like this:
Putting checks in place is important because:
* The amount and type of data the collector processes is specific to its environment.
* The collector's resource utilization is dependent on the configured processors.

Reply:
@barbnewrelic, I used your solution. This is a nice streamlined approach!